### PR TITLE
Alternatives to committing

### PR DESCRIPTION
**Pull request by ageorgou**
This is meant to address #16.

- Add an input option ~`push`~ `should_push` that lets you choose not to commit and push the diagram (true by default, to maintain current behaviour).
- As suggested in one of the comments, add an option to upload the diagram as an artifact of the workflow (not done by default).
- Also allow getting the SVG directly as an output of the action. This could be useful, for example, if you want to programmatically comment on a PR with the markdown of the diagram.

Note sure if this is overkill, comments very welcome.

### CODE REVIEW

**Review by Wattenberger — APPROVED** [review]
this is wonderful, thanks for the PR! I had no idea about artifacts, but they seem like a really flexible way to prevent from pushing every change

File: README.md [hunk 1]
```diff
@@ -38,32 +38,54 @@ excluded_globs: 'frontend/*.spec.js;**/*.{png,jpg};**/!(*.module).ts'
...
 Default: 9
 
-## `commit_message`
+### `push`
+
+Whether to make a new commit with the diagram and push it to the original repository.
+
+Should be a boolean value, i.e. `true` or `false`. See `commit_message` and `branch` for how to customise the commit.
+
+Default: `false`
```
Comment (Wattenberger) [inline README.md:61]: we should probably leave this as `true`, for backwards compatibility. I know there are projects using repo-visualizer@main

Comment (ageorgou) [reply README.md:61]: Oops, the default was indeed `true` but I miswrote it in the README! Will fix it.

File: README.md [hunk 2]
```diff
@@ -38,32 +38,54 @@ excluded_globs: 'frontend/*.spec.js;**/*.{png,jpg};**/!(*.module).ts'
...
 
-## `max_depth`
+### `max_depth`
 
 The maximum number of nested folders to show files within. A higher number will take longer to render.
 
 Default: 9
 
-## `commit_message`
+### `push`
```
[the remark this answers is not in the document]
Comment (ageorgou) [reply README.md]: Done!

**Comment by Wattenberger** [comment]
wonderful! thanks for the PR and quick updates! this is a great feature!

**Pull request merged**